vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -32,6 +32,14 @@
 
 #define VIC_MPI_ROOT 0
 
+/******************************************************************************
+ *  * @brief   This structure stores netCDF file name and corresponding nc_id
+ *   *****************************************************************************/
+typedef struct {
+    char    nc_filename[MAXSTRING];
+    int     nc_id;
+} nameid_struct;
+
```

Comment:
I tried a few things but nothing works out nicely. I think this is a bit of a design flaw here, but I don't see an easy fix. I still don't quite think it belongs where we put it, but we'll leave it be for now.